src/uct/ib/rdmacm/rdmacm_iface.c

```diff
@@ -521,10 +521,15 @@ static UCS_CLASS_INIT_FUNC(uct_rdmacm_iface_t, uct_md_h md, uct_worker_h worker,
         }
 
         if (rdma_bind_addr(self->cm_id, (struct sockaddr *)params->mode.sockaddr.listen_sockaddr.addr)) {
+            int err = errno;
```

Comment:
q: do we need `err`?

btw, can do:
```
status = ((err == EADDRNOTAVAIL) || (err == EADDRINUSE)) ? UCS_ERR_BUSY : UCS_ERR_IO_ERROR;
```

Reply:
yes, but better to save errno immediately to avoid to rewrite it incidentally adding some code before

Reply:
IMO classic if .. else easier to read with complex condition

Reply:
> yes, but better to save errno immediately to avoid to rewrite it incidentally adding some code before

I see

> IMO classic if .. else easier to read with complex condition

agree